biogeochem/FatesLitterMod.F90

```diff
@@ -424,6 +424,73 @@ function GetTotalLitterMass(this) result(total_mass)
     
     return
   end function GetTotalLitterMass
-
+  
+  ! =====================================================
+
+  subroutine adjust_SF_CWD_frac(dbh,ncwd,SF_val_CWD_frac,SF_val_CWD_frac_adj)
+     
+     !DESCRIPTION
+     !Adjust  the partitioning of struct + sawp into cwd pools based on 
+     !cohort dbh. This avoids struct and sapw from small cohorts going to
+     !fuel classes that are larger than their dbh. Here, struct + sapw are sent to the cwd
+     !class consistent with fuel class diameter thresholds (Fosberg et al., 1971;
+     !Rothermel, 1983)
+
+     !ARGUMENTS
+     real(r8), intent(in)               :: dbh !dbh of cohort [cm]
+     type(integer), intent(in)          :: ncwd !number of cwd pools
+     real(r8), intent(in)               :: SF_val_CWD_frac(:) !fates parameter specifying the
+                                                              !fraction of struct + sapw going
+                                                              !to each CWD class
+     real(r8), intent(out)              :: SF_val_CWD_frac_adj(:) !Updated cwd paritions
+     !
+     !LOCAL VARIABLES
+     !These diameter ranges are based on work by Fosberg et al., 1971 
+     
+     real(r8), parameter :: lb_max_diam        = 7.6 !max diameter [cm] for large branch
```

Comment:
Please add fuel classes next to comments on size to increase ability to cross reference with Fosberg and Rothermel.